doc/nrf/releases/release-notes-1.6.0.rst

```diff
@@ -0,0 +1,587 @@
+    * Added support for %CONEVAL AT command that can be used to evaluate the LTE radio signal state in a cell prior to data transmission.
+
+  * :ref:`serial_lte_modem` application:
+
+    * Fixed TCP/UDP port range issue (0~65535).
+    * Added AT#XSLEEP=2 to power off UART interface.
+    * Added support for the ``verbose``, ``uput``, ``mput`` commands and data mode to the FTP service.
+    * Added URC (unsolicited response code) to the FOTA service.
+    * Enabled all SLM services by default.
+    * Updated the HTTP client service code to handle chunked HTTP responses.
+    * Added data mode to the MQTT Publish service to support JSON-type payload.
+    * Added SMS support, to send/receive SMS in plain text.
+
+  * :ref:`at_cmd_parser_readme` library:
+
+    * Added support for parsing parameters of type unsigned int or unsigned short.
+
+  * :ref:`lib_spm` library:
+
+    * Added support for the nRF9160 pulse-density modulation (PDM) and inter-IC sound (I2S) peripherals in non-secure applications.
+    * Fixed an issue where SPM and the application could have incompatible FPU configurations, resulting in a HardFault.
+      Now, the application is free to use FPU regardless of SPM configuration.
+
+  * :ref:`gps_api` interface:
+
+    * Renamed gps_agps_request() to gps_agps_request_send().
+
+  * :ref:`agps_sample` sample:
+
+    * Added optional P-GPS support.
+    * Added :file:`overlay-pgps.conf` to enable P-GPS support.
+    * Added :file:`overlay-agps-pgps.conf` to enable A-GPS and P-GPS support.
+
+  * :ref:`aws_fota_sample` sample:
+
+    * Removed nRF Cloud support code, because ``fota_v1`` is no longer supported in nRF Cloud.
+    * Removed provisioning using :ref:`modem_key_mgmt` and :file:`certificates.h`, because this is not the recommended way of provisioning private certificates.
+    * Renamed the following Kconfig options:
+
+      * ``CONFIG_CLOUD_CERT_SEC_TAG`` renamed to :option:`CONFIG_CERT_SEC_TAG`.
+      * ``CONFIG_USE_CLOUD_CLIENT_ID`` renamed to :option:`CONFIG_USE_CUSTOM_CLIENT_ID`.
+      * ``CONFIG_CLOUD_CLIENT_ID`` renamed to ``CONFIG_CLIENT_ID``.
+      * ``CONFIG_NRF_CLOUD_CLIENT_ID_PREFIX`` renamed to ``CONFIG_CLIENT_ID_PREFIX``.
+
+  * :ref:`lib_aws_fota` library:
+
+    * Updated internal state handling and fault tolerance.
+
+  * :ref:`sms_readme` library:
+
+    * Updated to enable sending of SMS and decoding of received SMS payload.
+
+  * :ref:`liblwm2m_carrier_readme` library:
+
+    * Updated to v0.20.1.
+      See :ref:`liblwm2m_carrier_changelog` for details.
+
+nRF5
+====
+
+Bluetooth LE
+------------
+
+* Added:
+
+  * Support for serialization of the :ref:`zephyr:bt_gap` and the :ref:`zephyr:bluetooth_connection_mgmt` API.
+  * :ref:`ble_rpc_host` sample that enables support for serialization of the :ref:`zephyr:bt_gap` and the :ref:`zephyr:bluetooth_connection_mgmt`.
+  * Samples demonstrating the Direction Finding feature (available for the nRF52833 DK):
```

Comment:
@pdunaj  adding this as discussed, please verify

Reply:
I am ok, @ppryga any else to be added?

Reply:
```suggestion
* Samples demonstrating the direction finding feature in connectionless mode (available for the nRF52833 DK):
```
@mia-ko has checked BT core spec and the direction finding feature isn't written there with capitals. Also I suggest to state it explicitly that there is support for direction finding based on periodic advertising (connectionless), to avoid confusion for customers. Besides that I'm ok with it.

Reply:
done, thanks